docs/modify-volume.md

```diff
@@ -14,8 +14,8 @@ To use this feature, it must be enabled in the following places:
 - `VolumeAttributesClass` feature gate on `kube-apiserver` (consult your Kubernetes distro's documentation)
 - `storage.k8s.io/v1alpha1` enabled in `kube-apiserver` via [`runtime-config`](https://kubernetes.io/docs/tasks/administer-cluster/enable-disable-api/) (consult your Kubernetes distro's documentation)
```

Comment:
Can we change this line from `storage.k8s.io/v1alpha1` -> `storage.k8s.io/v1beta1`